gapis/api/vulkan/api/command_buffer_control.api

```diff
@@ -643,6 +651,7 @@ sub void dovkCmdExecuteCommands(ref!vkCmdExecuteCommandsArgs cmds) {
   leaveSubcontext()
 }
 
+@since("1.0")
```

Comment:
I would be happy to not include @since(1.0) and just assume that everything without an annotation is EITHER an extension or core functionality. (Just to trim down the number of things we have to add to EVERY single entry-point)